{{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/settings.py

```diff
@@ -168,3 +169,11 @@
 
 # This is needed to get a CRSF token in /admin
 ANON_ALWAYS = True
+
+USE_X_FORWARDED_HOST = config('USE_X_FORWARDED_HOST', default=True, cast=bool)
```

Comment:
What does this do? It needs a comment. 

Also, does this have anything to do with SecurityMiddleware? I don't see it mentioned in https://docs.djangoproject.com/en/1.9/ref/middleware/#module-django.middleware.security